atomicfu-transformer/src/main/kotlin/kotlinx/atomicfu/transformer/AtomicFUTransformer.kt

```diff
@@ -466,6 +470,7 @@ class AtomicFUTransformer(
         // Generates static VarHandle field
         private fun vhField(protection: Int, f: FieldInfo) {
             super.visitField(protection or ACC_FINAL or ACC_STATIC, f.fuName, VH_TYPE.descriptor, null, null)
+            super.visitField(protection or ACC_FINAL or ACC_STATIC, f.arrayLengthMethodHandleName, METHOD_HANDLE_TYPE.descriptor, null, null)
```

Comment:
I don't really see why it is even needed. We can directly access the underlying array's length without a method handle, cannot we?